app/views/components/widget/_fba2.js.erb

```diff
@@ -100,15 +89,14 @@ function FBAform(d, N) {
 		<%# inject the form interface for modal and custom-button-modal %>
 		if (this.options.deliveryMethod && (this.options.deliveryMethod === 'modal' || this.options.deliveryMethod === 'custom-button-modal')) {
 			this.dialogEl = d.createElement('div');
-			this.dialogEl.setAttribute("hidden", true);
-			this.dialogEl.setAttribute("aria-hidden", true);
-			this.dialogEl.setAttribute('class', 'fba-modal');
+			this.dialogEl.setAttribute('class', "<%= form.prefix "usa-modal" %> fba-modal");
+			this.dialogEl.setAttribute('id', this.modalId());
+			this.dialogEl.setAttribute('aria-labelledby', `fba-form-title-${this.options.formId}`);
+			this.dialogEl.setAttribute('aria-describedby', 'something');
```

Comment:
The modal code requires that you set `aria-describedby` to something non-empty. I'm not sure what to set it to in our case.